deploy-board/deploy_board/static/js/components/capacitycomponents.js

```diff
@@ -313,8 +327,17 @@ Vue.component("static-capacity-config", {
                     this.sizeWarning = getCapacityScaleDownAlertMessage(true, true);
                 }
             }
+
             this.imbalanceWarning = checkImbalance(this.placements, calculateImbalanceThreshold(sizeIncrease, this.placements.length));
             this.showImbalanceWarning = this.imbalanceWarning != '';
+
+            if ((!(this.terminationLimit === null)) && (-sizeIncrease > this.terminationLimit)) {
+                this.showTerminationError = true;
+                this.terminationError = getTerminationLimitAlertMessage(false);
+            } else if ((!(this.terminationLimit === null)) && (-sizeIncrease > this.terminationLimit - this.terminatingHostCount)) {
+                this.showTerminationError = true;
+                this.terminationError = getTerminationLimitAlertMessage(true);
+            }
```

Comment:
Seems like the logic to remove the warning when the user corrects the mistake is missing.
i.e. `this.showTerminationError = false;`

Reply:
it is in line 304